plugin/trino-delta-lake/README.md

```diff
@@ -0,0 +1,191 @@
+the corresponding product test environment:
+
+```
+testing/bin/ptl env up --environment 'EnvSinglenodeDeltaLakeOss'
+```
+
+
+## Delta Lake Databricks Product tests
+
+The Databricks Delta Lake implementation provides a richer functionality compared
+to the open source version.
+In order to setup a Databricks testing environment there are a few steps to be performed.
+At the time of this writing, Databricks is available on the following cloud providers:
+
+- AWS
+- Microsoft Azure
+- Google Cloud
+
+### Delta Lake Databricks on AWS
+
+Start by setting up a Databricks account via https://databricks.com/try-databricks and after
+filling your contact details, choose *AWS* as preferred cloud provider.
+
+Create an AWS S3 bucket to be used for storing the content of the Delta Lake tables managed
+by the Databricks runtime.
+
+Follow the guideline [Secure access to S3 buckets using instance profiles](https://docs.databricks.com/administration-guide/cloud-configurations/aws/instance-profiles.html)
+for allowing the Databricks cluster to access the AWS S3 bucket on which the Delta Lake tables are stored or AWS Glue
+table metastore.
+
+In order to make sure that the setup has been done correctly, proceed via Databricks Web UI to
+create a notebook on which a simple table could be created:
+
+```
+%sql
+CREATE TABLE default.test1 ( 
+    a_bigint BIGINT) 
+USING DELTA LOCATION 's3://my-s3-bucket/test1'
+```
+
+(Be sure to replace in the statement above `my-s3-bucket` with the name of your S3 bucket on which 
+the Databricks cluster has the ability to write to by the previously configured Instance profile).
+
+
+### Use AWS Glue Data Catalog as the metastore for Databricks Runtime
+
+[AWS Glue](https://aws.amazon.com/glue) is the metastore of choice for Databricks Delta Lake product tests
+on Trino because it is a managed solution which allows connectivity to the metastore backing the
+Databricks runtime from Trino as well while executing the product tests.
+
+Follow the guideline [Use AWS Glue Data Catalog as the metastore for Databricks Runtime](https://docs.databricks.com/data/metastores/aws-glue-metastore.html#configure-glue-data-catalog-as-the-metastore)
+for doing the setup on your Databricks Cluster.
+
+After performing successfully this step you should be able to perform any of the statements:
+
+```
+show databases;
+
+show tables;
+```
+
+The output of the previously mentioned statements should be the same as the one seen on the
+AWS Glue administration Web UI.
+
+
+### Create AWS user to be used by Trino for managing Delta Lake tables
+
+Trino needs a set of security credentials for successfully connecting to the AWS infrastructure
+in order to perform create/drop tables on AWS Glue and read/modify table content on AWS S3.
+
+Create via AWS IAM a user which has the appropriate policies for interacting with AWS.
+Below are presented a set of simplistic permission policies which can be configured on this
+user:
+
+`GlueFullAccess`
```

Comment:
it's not full access, is it?
(otherwise you'd use `"glue:*"` , or make the wildcard some other way, right?)

Reply:
As per document https://docs.databricks.com/data/metastores/aws-glue-metastore.html#configure-glue-data-catalog-as-the-metastore it doesn't need to be full access. I've modified the inline policy content.